pkg/boot/initrd.go

```diff
@@ -37,3 +39,29 @@ func CatInitrds(initrds ...io.ReaderAt) io.ReaderAt {
 		return bytes.NewReader(buf.Bytes()), nil
 	})
 }
+
+// CreateInitrd creates an initrd with the collection of files passed in.
+func CreateInitrd(files ...string) (io.ReaderAt, error) {
```

Comment:
It would good to add a test for this function.

Reply:
done!